### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by bcamacho**
### Description
Added two new pages with instructions to connect a Ledger device to Celo Web Wallet & Celo Terminal

### CODE REVIEW

**Review by marekolszewski — DISMISSED** [review]
This is great! What do you think of putting the CLI instructions on their own page as well and then linking to all three pages at the bottom of the current ledger page?

File: packages/docs/celo-holder-guide/connecting-ledger-celo-terminal-wallet.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Connect Ledger to Celo Terminal
```
Comment (critesjosh) [inline packages/docs/celo-holder-guide/connecting-ledger-celo-terminal-wallet.md:1]: I removed the `<br>` tags in this file, it was making the page render strangely.

File: packages/docs/celo-holder-guide/connecting-ledger-celo-web-wallet.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Connect Ledger to Celo Web Wallet
+
+## Step 1
+</br>
+### Navigate to [https://celowallet.app/setup](https://celowallet.app/setup)
+</br>
+
+![https://Celowallet.app/setup](https://user-images.githubusercontent.com/2653576/130851740-742bb884-7ab7-465f-9d33-861ac6a5e909.png "image_tooltip")
```
Comment (critesjosh) [inline packages/docs/celo-holder-guide/connecting-ledger-celo-web-wallet.md:8]: It looks like the pictures are a little out of sync here. Should this picture be a part of step 2?

File: packages/docs/celo-holder-guide/connecting-ledger-celo-web-wallet.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Step 2
+
+</br>
+### Select “Use existing wallet”
+</br>
+
+![https://celowallet.app/setup/existing](https://user-images.githubusercontent.com/2653576/130851953-92ab2c36-01d3-493c-95d4-dce1d39f05fe.png "image_tooltip")
```
Comment (critesjosh) [inline packages/docs/celo-holder-guide/connecting-ledger-celo-web-wallet.md:14]: and this picture should be part of step 3

File: packages/docs/celo-holder-guide/connecting-ledger-celo-web-wallet.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Connect Ledger to Celo Web Wallet
+
+## Step 1
+</br>
```
Comment (critesjosh) [inline packages/docs/celo-holder-guide/connecting-ledger-celo-web-wallet.md]: suggest removing these breaks from this file as well.

File: packages/docs/celo-holder-guide/ledger.md [hunk 5]
```diff
@@ -53,157 +52,8 @@ Press both buttons on the device at the same time to continue.
 
 The Celo app is now ready for use and you should see `Application is ready` on the screen.
 
-## Setup Instructions
-### Install the Celo CLI
+#### Connect your Leger to Celo applications
```
Comment (marekolszewski) [inline packages/docs/celo-holder-guide/ledger.md:55]: After this heading, what do you think of an intro sentence like the following:

Now that your ledger is all set up, you can start using it with the following wallets:

File: packages/docs/celo-holder-guide/connecting-ledger-celo-terminal-wallet.md (hunk 1 above)
Comment (bcamacho) [reply packages/docs/celo-holder-guide/connecting-ledger-celo-terminal-wallet.md:1]: I used a local markdown editor which caused a conflict as GitHub renders different. The lesson I learned, verify with Github preview.

**Review by bcamacho — COMMENTED** [review]
Resolved errors from feedback with PR. @critesjosh please review.

File: packages/docs/celo-holder-guide/ledger.md (hunk 5 above)
Comment (bcamacho) [reply packages/docs/celo-holder-guide/ledger.md:55]: Great suggestion! Added the modification from your feedback.

**Review by critesjosh — APPROVED** [review]
lgtm!

**Pull request merged**